x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_helpers.ts

```diff
@@ -211,26 +212,31 @@ function getVisualizationSuggestions(
   isFromContext?: boolean,
   activeData?: Record<string, Datatable>
 ) {
-  return visualization
-    .getSuggestions({
-      table,
-      state: currentVisualizationState,
-      keptLayerIds: datasourceSuggestion.keptLayerIds,
-      subVisualizationId,
-      mainPalette,
-      isFromContext,
-      activeData,
-    })
-    .map(({ state, ...visualizationSuggestion }) => ({
-      ...visualizationSuggestion,
-      visualizationId,
-      visualizationState: state,
-      keptLayerIds: datasourceSuggestion.keptLayerIds,
-      datasourceState: datasourceSuggestion.state,
-      datasourceId: datasourceSuggestion.datasourceId,
-      columns: table.columns.length,
-      changeType: table.changeType,
-    }));
+  try {
```

Comment:
This is catching errors in visualization suggestions, but not in the data source - could we cover https://github.com/elastic/kibana/blob/071a8663911a9a8bb24b504226484b43f286c6c9/x-pack/plugins/lens/public/editor_frame_service/editor_frame/suggestion_helpers.ts#L86 in the same way?